### ISSUE TITLE
[Windows 32 Bit]  jack2-win-32-v1.9.15.exe is not a win32 valid application

### ISSUE THREAD

**Issue by E-Gwen** [jackaudio/jack2#645]
Hi,

When I try to install J.A.C.K 1.9.15, I get this error message :

"D:\jack2-win-32-v1.9.15.exe n'est pas une application win32 valide"

    ("D:\jack2-win-32-v1.9.15.exe is not a win32 valid application")

As I want to try Carla 2.2 with no SSE build and Ardour 6.3 with Windows XP, don"t you think it will be appropriate to make a jack release compatible with this OS ?

Notice that J.A.C.K. 1.9.11 32 bit version is running fine.

Best regards.

E-Gwen.

**Comment by falkTX** [comment]
so this is related to Windows XP perhaps?
I was finding it weird that you get that message, but that might be the reason.

**Comment by E-Gwen** [comment]
Yes it is.                                                                                                                                                                                                                                                                                                                                        Envoyé de mon smartphone BlackBerry 10 sur le réseau SFR.                                                                                                                                                                                                                De: Filipe CoelhoEnvoyé: jeudi 15 octobre 2020 16:56À: jackaudio/jack2Répondre à: jackaudio/jack2Cc: Gwen; AuthorObjet: Re: [jackaudio/jack2] [Windows 32 Bit]  jack2-win-32-v1.9.15.exe is not a win32 valid application (for Windows XP) (#645)
so this is related to Windows XP perhaps?
I was finding it weird that you get that message, but that might be the reason.

—You are receiving this because you authored the thread.Reply to this email directly, view it on GitHub, or unsubscribe.
[
{
"@context": "http://schema.org",
"@type": "EmailMessage",
"potentialAction": {
"@type": "ViewAction",
"target": "https://github.com/jackaudio/jack2/issues/645#issuecomment-709382028",
"url": "https://github.com/jackaudio/jack2/issues/645#issuecomment-709382028",
"name": "View Issue"
},
"description": "View this Issue on GitHub",
"publisher": {
"@type": "Organization",
"name": "GitHub",
"url": "https://github.com"
}
}
]

**Comment by falkTX** [comment]
If the binaries work on windows 10, but not on xp, I think that is it for this issue.. 
no one should be using XP at this point, like.. please, really, no

**Comment by E-Gwen** [comment]
All right.But I am.Who is using Windows 10 32bit version ? No one should neither.So I have to expect that someone will try to install this version.How do you want us to try Carla 2.2 with no sse-build  ‎correctly, wether, as you say yourself, we don't use at least J.A.C.K. 1.9..13 ?Maybe with a very last one release of JACK for Windows XP ? Cheers.                                                                                                                                                                                                   Envoyé de mon smartphone BlackBerry 10 sur le réseau SFR.                                                                                                                                                                                                                De: Filipe CoelhoEnvoyé: jeudi 15 octobre 2020 17:18À: jackaudio/jack2Répondre à: jackaudio/jack2Cc: Gwen; AuthorObjet: Re: [jackaudio/jack2] [Windows 32 Bit]  jack2-win-32-v1.9.15.exe is not a win32 valid application (for Windows XP) (#645)
If the binaries work on windows 10, but not on xp, I think that is it for this issue..
no one should be using XP at this point, like.. please, really, no

—You are receiving this because you authored the thread.Reply to this email directly, view it on GitHub, or unsubscribe.
[
{
"@context": "http://schema.org",
"@type": "EmailMessage",
"potentialAction": {
"@type": "ViewAction",
"target": "https://github.com/jackaudio/jack2/issues/645#issuecomment-709396795",
"url": "https://github.com/jackaudio/jack2/issues/645#issuecomment-709396795",
"name": "View Issue"
},
"description": "View this Issue on GitHub",
"publisher": {
"@type": "Organization",
"name": "GitHub",
"url": "https://github.com"
}
}
]

**Comment by HaHeho** [comment]
Give the man a break. He just made all these installers available. We're lucky to have them in the first place.

This is open source software. No reason to complain, just make it better yourself!

When running an exotic system (and I think it is fair to call WinXP 32bit exotic these days) you cannot rely on software support. You will have to be the expert (and probably build) yourself.

What is the reason for running that system and having to update Jack?
